packages/modelviewer.dev/scripts/ci-before-deploy.sh

```diff
@@ -53,6 +53,13 @@ DEPLOYABLE_STATIC_FILES=( \
   shared-assets/models/glTF-Sample-Models/2.0/Suzanne \
   shared-assets/models/glTF-Sample-Models/2.0/SpecGlossVsMetalRough \
   shared-assets/environments \
+  node_modules/@google/model-viewer-space-opera/node_modules/web-animations-js/web-animations-next-lite.min.js \
+  node_modules/@google/model-viewer-space-opera/node_modules/js-beautify/js/lib/beautify-html.js \
+  node_modules/@google/model-viewer-space-opera/node_modules/js-beautify/js/lib/beautify-css.js \
+  node_modules/@google/model-viewer-space-opera/index.html \
+  node_modules/@google/model-viewer-space-opera/node_modules/@google/model-viewer-shared-assets/environments/spruit_sunrise_1k_HDR.hdr \
```

Comment:
Not crazy about this duplication of HDRs. Not sure about a better solution...for resolving these paths.

Reply:
This gets back to the earlier comment I left you about paths. I believe there is a shared-assets symlink in space-opera already, like the rest of our packages. I think you can use that instead of the module path. Can you see if there's a way following how the other packages reference shared-assets?

Reply:
ok lemme try referring to shared_assets directly in both settings...

Reply:
ok this seems to work fine.